presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcModule.java

```diff
@@ -85,4 +85,14 @@ public static void bindProcedure(Binder binder, Class<? extends Provider<? exten
     {
         procedureBinder(binder).addBinding().toProvider(type).in(Scopes.SINGLETON);
     }
+
+    public static Multibinder<TablePropertiesProvider> tablePropertiesProviderMultiBinder(Binder binder)
```

Comment:
Remove `Multi` from the name for consistency with `bindSessionPropertiesProvider`

Reply:
How about `tablePropertiesBinder`?

Reply:
Renamed it as `bindTablePropertiesProvider`

Reply:
We have `JdbcModule#sessionPropertiesProviderBinder`, so `tablePropertiesProviderBinder` would be consistent.